src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -166,18 +183,31 @@ const renderJiraCloudAndEnterpriseServer = async (res: Response, req: Request):
 	res.render("jira-configuration-new.hbs", {
 		host: jiraHost,
 		gheServers: groupedGheServers,
-		ghCloud: { successfulConnections, failedConnections },
-		hasCloudAndEnterpriseServers: !!((successfulConnections.length || failedConnections.length) && gheServers.length),
-		hasCloudServers: !!(successfulConnections.length || failedConnections.length),
+		ghCloud: { successfulCloudConnections, failedCloudConnections },
+		hasCloudAndEnterpriseServers: !!((successfulCloudConnections.length || failedCloudConnections.length) && gheServers.length),
+		hasCloudServers: !!(successfulCloudConnections.length || failedCloudConnections.length),
 		hasConnections: !!(installations.total || gheServers?.length),
 		APP_URL: process.env.APP_URL,
 		csrfToken: req.csrfToken(),
 		nonce
 	});
+
+	const successfulServerConnections = gheServersWithConnections
+		.reduce((acc, obj) => acc + obj.successfulConnections?.length, 0);
+	const allSuccessfulConnections = [...successfulCloudConnections, ...gheServersWithConnections];
+	const completeConnections = allSuccessfulConnections.filter(connection => connection.syncStatus === "FINISHED");
+
 	sendAnalytics(AnalyticsEventTypes.ScreenEvent, {
 		name: AnalyticsScreenEventsEnum.GitHubConfigScreenEventName,
 		jiraHost,
-		connectedOrgCount: installations.total + gheServers.length
+		connectedOrgCountCloudCount: successfulCloudConnections.length,
+		connectedOrgCountServerCount: successfulServerConnections,
+		totalOrgCount: successfulCloudConnections.length + successfulServerConnections,
+		failedCloudBackfillCount: countStatus(successfulCloudConnections, "FAILED"),
```

Comment:
This is a bit confusing. Can there be any `successfulCloudConnection` whose status is `FAILED`? 

All these connections come from `installations` in the method `getConnectionsAndInstallations`. Amongst these `installations`, the `successful` and `failed` ones are already separated. So, can there be a status `FAILED` connection in the already filtered out `successfulCloudConnection`? 

Shouldn't this be `failedCloudConnections.length`?